core/src/raw/ops.rs

```diff
@@ -601,6 +601,7 @@ pub struct OpWrite {
     cache_control: Option<String>,
     executor: Option<Executor>,
     if_none_match: Option<String>,
+    if_not_exist: Option<bool>,
```

Comment:
We can simply use `bool` here, as we don't have special handling for `None`.

Reply:
By the way, please add `if_not_exists` instead of `if_not_exist` to align with our existing API `exists`

Reply:
Fixed.